changelog/16274.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
@austingebauer could you please update this changelog note to either use the "new feature" format, or move this changelog note to a different section of the changelog?

Reply:
Will do, @mladlow 👍